filebrowser/static/filebrowser/js/FB_TinyMCEv4.js

```diff
@@ -5,11 +5,14 @@
  */
 var FileBrowserDialogue = {
     fileSubmit : function (FileURL) {
-        parentWin = (!window.frameElement && window.dialogArguments) || opener || parent || top;
+        parentWin = (!window.frameElement && window.dialogArguments) || parent;
```

Comment:
> had to remove "opener" and "top" from var parentWin, because they will become NULL after called once

there is no problem with them becoming null.
```
(!window.frameElement && window.dialogArguments) || opener || parent || top
```

means it will use the first not falsy (undefined or null) value and just skip the rest.